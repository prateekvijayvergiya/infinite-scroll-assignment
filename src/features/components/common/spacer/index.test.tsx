import Component from './'
import makeSnapshot from './../../../testing/makeSnapshot'

describe('Spacer', () => {
  it('renders correctly', () => {
    const props = {
      spacing: 1,
    }
    makeSnapshot(<Component {...props} />)
  })
})
