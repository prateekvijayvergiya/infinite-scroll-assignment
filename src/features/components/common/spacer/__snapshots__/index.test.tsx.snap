// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer renders correctly 1`] = `
<div
  className="makeStyles-root makeStyles-root"
/>
`;
